presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -802,6 +802,9 @@ public void testBinaryFunctions()
         // crc32.
         assertQuery("SELECT crc32(cast(comment as varbinary)) FROM orders");
 
+        // from_base32, to_base32.
+        assertQuery("SELECT from_base32(to_base32(cast(comment as varbinary))) FROM orders");
```

Comment:
Would you please verify what the comment column contents look like and share it here. Just curious.
How many rows and length of each row.